example.csharp/addons/tiltfive/T5Manager.cs

```diff
@@ -0,0 +1,111 @@
+using Godot;
+using System;
+
+[GlobalClass]
+public partial class T5Manager : Node3D, T5ManagerInterface
```

Comment:
This now extends from Node3D where-as the gdscript version extends from Node. I was able to make TiltFiveTools work by changing the base type to match, but could we get away with just a Node?